product_medical/views/product_template.xml

```diff
@@ -19,22 +19,22 @@
                         <group name="conformity_left" string="Categories">
                             <field name="medical_class_id" />
                             <field name="medicine_category_id" />
-                            <field name="ppe_category_id" />
```

Comment:
We should rename the `conformity` page to `medical_confirmity` still to avoid conflicts with other modules